polaris/dataset/_dataset.py

```diff
@@ -485,7 +485,9 @@ def _split_index_from_path(self, path: str) -> Tuple[str, Optional[int]]:
             index = index.split(":")
 
             if len(index) == 1:
-                index = int(index[0])
+                index = index[0]
+                if index.isdigit():
+                    index = int(index)
```

Comment:
**Suggestion**:  I found this piece of code a little confusing. 

After looking at it for a bit, I realize it's because currently the "pointer" looks something like: 
```
pdb#tutorial
```

Whereas based on the name of the variables, I would expect
```
pdb/tutorial/
```

We should only use `#` when we want to index an array, not when we want to access a group.

I actually think that without these lines of code, it should work with the path I suggested.